docs/reference/mount.md

```diff
@@ -143,15 +178,43 @@ ls -l "./pets/dogs/"
 find ./pets -name "*.small.jpg"
 open -a Preview "./pets/dogs/golden_retrievers/cute.jpg"
 everest umount "./pets"
-
 ```
-#### Working with Data Locally
+
+##### Working with Data Locally
 Mount the remote lakeFS server and use all familiar tools without changing the workflow.
 ```bash
-everest mount "lakefs://image-repo/main/datasets/pets/" "./pets"
-pytorch_train.py --input "./pets"
+everest mount lakefs://image-repo/main/datasets/pets/ ./pets
+pytorch_train.py --input ./pets
 duckdb "SELECT * FROM read_parquet('pets/labels.parquet')"
-everest umount "./pets"
+everest umount ./pets
+```
+
+### Write Mode
+
+##### Changing Data Locally
+Mount the remote lakeFS server in write mode and change data locally.
```

Comment:
Add URL to the semantics here as well

Reply:
I'm not sure I understand this comment.
What URL?

Reply:
The second page you created with semantics

Reply:
Again, not sure what do you mean
Add URL of what? in which context?